bin/update-package-json.mts

```diff
@@ -3,7 +3,7 @@ import { resolve } from 'node:path';
 
 import chalk from 'chalk';
 
-import { type Package, packages } from './packages.mjs';
+import { packages, type Package } from './packages.mjs';
```

Comment:
@NullVoxPopuli this

Reply:
https://github.com/glimmerjs/glimmer-vm/pull/1507